README.md

```diff
@@ -1,2 +1,155 @@
-# melita-io-batch
-Melita API module to execute batch command on multiple devices
+# Melita IO
+
+|                |                                  |
+| -------------- | -------------------------------- |
+| Name           | Melita IO Batch                  |
+| Version        | v1.0.0                           |
+| Dockerhub Link | [weevenetwork/melita-io-batch]() |
+| Authors        | Mesud Pasic                      |
+
+- [MQTT Ingress](#melita-io-batch)
+  - [Description](#description)
+  - [Features](#features)
+  - [Environment Variables](#environment-variables)
+    - [Module Specific](#module-specific)
+    - [Set by the weeve Agent on the edge-node](#set-by-the-weeve-agent-on-the-edge-node)
+  - [Dependencies](#dependencies)
+
+## Description
+
+Melita.IO API module.
+
+## Features
+
+- Parsing Melita.io data for thermostat
+- Sends data to next module service via REST API
+
+## Environment Variables
+
+- INGRESS_HOST
+- INGRESS_PORT
+- EXECUTE_SINGLE_COMMAND
+- SINGLE_COMMAND
+- DEVICE_EUI_LIST
```

Comment:
Replace it with table of Environment Variables set in the UI:

| Environment Variables | type | Description |
| --- | --- | --- |
| EXECUTE_SINGLE_COMMAND | string | yes/no to specify if module supports only executing single command |
| SINGLE_COMMAND | string | if EXECUTE_SINGLE_COMMAND=no, then users will have to provide in request "command" parameter with command name |
| MELITA_API_URL | string | Melita.io API URL |
| AUTHENTICATION_API_KEY | string | API key for authentication |
| ERROR_URL | string | URL for passing errors during request to melita.io API |
| DEVICE_EUI_LIST | string | List of device EUIs seperated by comma |

Reply:
removed these two from the list
- INGRESS_HOST
- INGRESS_PORT